tools/extras/install_tensorflow_cc.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 #export JAVA_HOME=/LOCATION_ON_YOUR_MACHINE/java/jdk1.8.0_121
+export JAVA_HOME=/export/b02/hxu/TensorFlow/java/jdk1.8.0_121
```

Comment:
I don't think this is a great idea. You should just check that JAVA_HOME isn't empty string instead, with `[ ! -z $JAVA_HOME ]`